op-service/sources/rollupclient.go

```diff
@@ -22,20 +23,37 @@ func NewRollupClient(rpc client.RPC) *RollupClient {
 
 func (r *RollupClient) OutputAtBlock(ctx context.Context, blockNum uint64) (*eth.OutputResponse, error) {
 	var output *eth.OutputResponse
+	// If you provide `nil` to rpc.CallContext, it will return `nil` so we have to check the returned value
+	// (even though we know it is not nil) to satisfy nilaway
```

Comment:
Hmm so this looks like we have to add complexity just to satisfy nilaway, but it's not really an issue. If we really want to use `nilaway`, is there a way to instead tell it to ignore this function call?